packages/orbit-components/src/Card/Card.stories.tsx

```diff
@@ -339,12 +340,19 @@ export const CardWithMixedSections: Story = {
       <CardSection title={sectionTitle} description={sectionDescription}>
         Section Content
       </CardSection>
+      <CardSection onClick={onClick} title={sectionTitle} description={sectionDescription}>
```

Comment:
I added a variant with `CardSection` with `onClick` function, I haven't found this variant in SB.

Reply:
I added these to the story so I can check all cases, so all my comments in the review are based on also these

```
<Card>
        <CardSection
          actions={<ButtonLink onClick={handleClick}>Open</ButtonLink>}
          onClick={onClick}
          expandable={false}
          title={sectionTitle}
        >
          Section Content with onClick
        </CardSection>
        <CardSection
          actions={<ButtonLink onClick={handleClick}>Open</ButtonLink>}
          onClick={onClick}
          expandable
          title={sectionTitle}
        >
          Section Content with onClick
        </CardSection>
</Card>
```